### PR TITLE
Calculate

### PR DESCRIPTION
**Pull request by KamilFinanseOrazProgramowanie**
Solution

### CODE REVIEW

File: module1/homework/calculate/calculate.hpp [hunk 1]
```diff
@@ -2,6 +2,30 @@
 #include <string>
 
 std::string calculate(const std::string& command, int first, int second) {
-    // TODO: Implement your solution here and return proper value
+    int result = 0;
+      
+    if (command == "add"){
+        result = first + second;
+        return std::to_string(result);
```
Comment (TomaszEichler) [inline module1/homework/calculate/calculate.hpp:9]: You can write it in shorter form : `return std::to_string(first + second)`

File: module1/homework/calculate/calculate.hpp [hunk 2]
```diff
@@ -2,6 +2,30 @@
...
+        if (second == 0){
+        return std::string("Division by Zero");
+        }
+        result = first / second;
+        return std::to_string(result);
+      }
+    else {
+        return "Invalid data";
+    }
     return "";
```
Comment (TomaszEichler) [inline module1/homework/calculate/calculate.hpp:29]: you just need here `return "Invalid data"`. `else` is unnecessary and last `return ""` also

File: module1/homework/calculate/calculate.hpp [hunk 3]
```diff
@@ -2,6 +2,30 @@
...
+        if (second == 0){
+        return std::string("Division by Zero");
+        }
+        result = first / second;
+        return std::to_string(result);
+      }
+    else {
+        return "Invalid data";
+    }
+    return 0;
```
Comment (MechaniQ1) [inline module1/homework/calculate/calculate.hpp]: This return is unnecessary, because function will always return value from if, elseif or else

**Comment by ziobron** [comment]
{ "requirement_ids": [1119, 1429], "points": [2, 5] }

**Pull request closed without merging**